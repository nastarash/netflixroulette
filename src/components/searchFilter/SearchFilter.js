import React from 'react';

function SearchFilter() {
  return (
    <form>
      <p>SEARCH BY</p>
      <div>
        <input
          id="toggle-on"
          className="toggle toggle-left"
          name="toggle"
          value="false"
          type="radio"
          //checked
        />
        <label htmlFor="toggle-on" className="btn">
          Title
        </label>
        <input
          id="toggle-off"
          className="toggle toggle-right"
          name="toggle"
          value="true"
          type="radio"
        />
        <label htmlFor="toggle-off" className="btn">
          Genre
        </label>
      </div>
    </form>
  );
}

export default SearchFilter;
