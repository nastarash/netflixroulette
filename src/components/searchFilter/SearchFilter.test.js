import React from 'react';
import renderer from 'react-test-renderer';
import SearchFilter from './SearchFilter';

it('render correctly', () => {
  const AppComponent = renderer.create(<SearchFilter />).toJSON();
  expect(AppComponent).toMatchSnapshot();
});
