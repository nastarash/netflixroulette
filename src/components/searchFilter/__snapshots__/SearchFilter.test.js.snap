// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<form>
  <p>
    SEARCH BY
  </p>
  <div>
    <input
      className="toggle toggle-left"
      id="toggle-on"
      name="toggle"
      type="radio"
      value="false"
    />
    <label
      className="btn"
      htmlFor="toggle-on"
    >
      Title
    </label>
    <input
      className="toggle toggle-right"
      id="toggle-off"
      name="toggle"
      type="radio"
      value="true"
    />
    <label
      className="btn"
      htmlFor="toggle-off"
    >
      Genre
    </label>
  </div>
</form>
`;
